pom.xml

```diff
@@ -194,6 +194,11 @@
             <artifactId>pipeline-utility-steps</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.googlecode.owasp-java-html-sanitizer</groupId>
```

Comment:
Seems slightly awkward to have a plugin dependency to only get to the library, when it's not an actual library plugin. Unsure.

(I would discourage using markup formatter plugin functionality, otherwise behavior of this feature depends on what's allowed for user-provided markup. That caused some trouble in uno-choice in the past.)

Reply:
> Seems slightly awkward to have a plugin dependency to only get to the library, when it's not an actual library plugin. Unsure.

it is at least what [timestamper](https://github.com/jenkinsci/timestamper-plugin/blob/ce7a5ac5422af70ab9d96638e333f698137c6920/src/main/java/hudson/plugins/timestamper/format/TimestampFormatUtils.java#L17-L22) [does](https://github.com/jenkinsci/timestamper-plugin/blob/ce7a5ac5422af70ab9d96638e333f698137c6920/pom.xml#L78-L81) :-/

so there are now 3 plugins that use the library which is normally the time to create an API plugin then...?
The security aspects of that esp for the markup formatter make me a little squeamish though...

Reply:
So conclusion here is to not rely on the markup-formatter, but instead to create another jenkins ticket to create a API plugin? When that is up and sorted we can switch other plugins over to using the API plugin. Give me a thumbs up on this and I will create the jira ticket 🙂

Reply:
I don't think we have a consensus :)